Misc/NEWS.d/next/Core_and_Builtins/2025-12-24-11-39-59.gh-issue-143135.3d5ovx.rst

```diff
@@ -0,0 +1 @@
+Fix :data:`sys.flags.inspect` not being set when :envvar:`PYTHONINSPECT` is 0.
```

Comment:
It is unclear -- was it not set when PYTHONINSPECT is 0, and it it is now set, or it is now not set?

Make clear that its value now consistent with the interpreter behavior asnd the documentation.